tensorflow/tools/docs/gen_cc_md.py

```diff
@@ -275,7 +275,8 @@ def get_type(self):
     return self.type
 
   def get_md_filename(self):
-    return self.get_type() + anchorize(self.get_short_name()) + '.md'
+    deftype = self.type[0].upper() + self.type[1:]
```

Comment:
I just made that same change. I think that's the second commit we're crossing on just today. :)

My change is upstream, it'll get to github in the next push out of google.